docker/nginx/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM alpine:3.14
+
+RUN apk add --no-cache \
+        nginx nginx-mod-http-vts \
+        curl ca-certificates certbot-nginx gettext tzdata \
+    && ln -sf /dev/stdout /var/log/nginx/access.log \
+    && ln -sf /dev/stderr /var/log/nginx/error.log \
+    && mkdir /docker-entrypoint.d \
+    && mkdir -p /usr/share/nginx/html \
+    && touch /usr/share/nginx/html/index.html
+
+# entrypoint scripts copied from official nginx site and adapted
+# to alpine nginx version
+COPY docker-entrypoint.sh /
+COPY entrypoint/*.sh /docker-entrypoint.d/
```

Comment:
This is not like https://github.com/nginxinc/docker-nginx/blob/master/Dockerfile-alpine.template at all?

Reply:
It is not. While initially we had an idea to base it off the nginx, it turned out they compile their version. That blocks any version compiled by alpine guys as module version of nginx api mismatch the nginx version. So finally I have decided to adapt nginx included within alpine with some sugar as mentioned in line 12 :)